cmd/releasego/report.go

```diff
@@ -105,5 +105,5 @@ func handleReport(p subcmd.ParseFunc) error {
 
 	log.Printf("Reporting %#v\n", s)
 	ctx := context.Background()
-	return buildreport.Update(ctx, owner, name, *pat, *issue, s)
+	return buildreport.Update(ctx, owner, name, *gitHubAuthFlags.GitHubPat, *issue, s)
```

Comment:
This means the command will claim to support app auth, but really it only will accept a PAT.

I would recommend reverting everything other than the usage that is actually being set up in the yaml to keep the PR tidy while working on the factoring.

Reply:
IMO if WIP makes it hard to review a PR (or update a PR in response to review) or makes any visible change to functionality, it should generally not be committed. I don't see the upside to keeping it in this case.

Reply:
I pushed a targeted revert--if any other commands are known not to be complete, I think they should be reverted too.